src/status_im/contexts/wallet/account/edit_account/view.cljs

```diff
@@ -35,6 +36,8 @@
 (defn view
   []
   (let [edited-account-name (reagent/atom nil)
+        name-error          (reagent/atom nil)
```

Comment:
Is there any particular reason to prefer atom over state here?

Reply:
We already have atoms for the state in these screens, and I didn't plan to rewrite the component, so to keep  the consistency I used atoms